drafts/WEP0002-slack-community-moderation-procedures.rst

```diff
@@ -0,0 +1,142 @@
+
+  Hi Mildred,
+
+  I'm Janine, one of the moderators here in the Write the Docs Slack community. In light of your comments above, I wanted to remind you to please keep the Code of Conduct in mind - be civil and treat others with respect. Please let me know if you have any questions. Thanks!
+
+A moderator might opt for a less formal reply, to further lower the barrier for action:
+
+  Hey Mildred, this is a friendly mod reminder to please respect others, even when you don't agree with them. If you have any questions, please feel free to DM me. Thanks!
+
+Conversely, depending on the context, a moderator might decide that their message needs a more direct tone:
+
+  Mildred, I'm one of the mods here, and that tone isn't acceptable. I'm asking you to please play nice, or take a break from this conversation.
+
+An example of a more formal warning in DM format:
+
+  Hi Mildred,
+
+  I'm Janine, one of the moderators here in the Write the Docs Slack community. I wanted to let you know that your conversation in #general earlier today violated our Code of Conduct rules, specifically the one that asks all users to remain respectful of each other at all times.
+
+  While I'm sure you were operating in good faith, we want to ensure that all users in this Slack community feel comfortable engaging with each other, and this means making sure that we all act civilly. To this end, I wanted to give you a reminder that we need you to follow those rules at all times.
+
+  If you have any questions, please don't hesitate to reach out and ask - the moderators are here to help. Thanks!
+
+2. Formal warning
+________________
+If a user continues to violate the Code of Conduct, a moderator can give them a more stern warning about their conduct. Example wording:
+
+  Hi Mildred,
+
+  I wanted to check in with you again regarding your posts in #general. As discussed before, I want to make it clear that all members of this community are required to follow the Code of Conduct rules at all times, with no exceptions.
+
+  If this continues to be a problem, the moderation team will have no choice but to consider further action up to and including removing you from the Write the Docs community the next time you violate the rules of this community. Because this is always a last resort that the moderation team wants to prevent wherever possible, I wanted to give you a fair warning first and the opportunity for you to ask me questions if you need to. Thanks!
```

Comment:
```suggestion
  This is a formal warning from the moderation team. If this continues to be a problem, the moderation team will have no choice but to consider further action up to and including removing you from the Write the Docs community the next time you violate the rules of this community. Because this is always a last resort that the moderation team wants to prevent wherever possible, I wanted to give you a fair warning first and the opportunity for you to ask me questions if you need to. Thanks!
```